index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+require 'src/db.php';
+
+require 'src/functions.php';
+
+?>
+
+<!DOCTYPE html>
+<html lang="en" lang="vi">
+<head>
+	<title>Bookshelf</title>
```

Comment:
indentation here is too far out

Reply:
seems fine here but i could have got wrong end of stick